zeppelin-web/app/scripts/app.js

```diff
@@ -95,9 +90,11 @@ angular
     'puElasticInput',
     'xeditable'
   ])
-  .filter('breakFilter', function() {
+  .filter('breakFilter', function () {
```

Comment:
well, here's a link to a _very common_ airbnb styleguide that recommends otherwise
https://github.com/airbnb/javascript#18.3

what I'm trying to say here is that since Zeppelin hasn't have its own styleguide yet, let's not nitpick on style. there are larger problems to solve :)